src/main/resources/defaults/data/Banners.json

```diff
@@ -34,49 +34,33 @@
 		"gachaType": 301,
 		"scheduleId": 903,
 		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A097",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A097_TITLE",
+		"prefabPath": "GachaShowPanel_A103",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A081_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1071],
+		"rateUpItems1": [1073],
+		"rateUpItems2": [1073]
 		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
-	},
-	{
-		"comment": "Character Event Banner 2",
-		"gachaType": 400,
-		"scheduleId": 923,
-		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A098",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A036_TITLE",
-		"costItemId": 223,
-		"beginTime": 0,
-		"endTime": 1924992000,
-		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1022],
-		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
+		"weights5":[[1,75],[73,150],[90,10000]],
 	},
 	{
 		"comment": "Weapon Event Banner",
 		"gachaType": 302,
 		"scheduleId": 913,
 		"bannerType": "WEAPON",
-		"prefabPath": "GachaShowPanel_A099",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A021_TITLE",
+		"prefabPath": "GachaShowPanel_A105",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A099_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9997,
-		"rateUpItems4":[12415, 11405, 13407, 14403, 15401],
-		"rateUpItems5": [13511, 15503],
+		"rateUpItems4": [14511]
+		"rateUpItems5": [14511,15509],
 		"fallbackItems5Pool1": [],
 		"weights4": [[1,600], [7,600], [8,6600], [10,12600]],
-		"weights5": [[1,100], [62,100], [73,7800], [80,10000]],
+		"weights5":[[1,10000],[73,10000],[90,10000]],
```

Comment:
Why are you changing the weights? Why hard pity at 73? Why an additional curve point at 90 when 10000 has already been placed?